ui/packages/ui/src/app/components/ApplicationErrorPage.tsx

```diff
@@ -8,60 +8,72 @@ import {
   PageSection,
   PageSectionVariants,
   Text,
-  Title,
+  Title
 } from "@patternfly/react-core";
 import { ExclamationTriangleIcon } from "@patternfly/react-icons";
-import { useState } from "react";
 import * as React from "react";
+import { useState } from "react";
+import { useTranslation } from 'react-i18next';
 import "./ApplicationErrorPage.css";
 
 export interface IApplicationErrorPageProps {
   icon?: React.ComponentType<any>;
-  title?: string;
```

Comment:
The idea to have these optional props was to make this component configurable and use in diff error causing scenarios, whereas the default case will be used in `WithErrorBoundary` component